src/Datadog.Trace/RandomNumberGenerator.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// RandomNumberGenerator implementation is the 64-bit random number generator based on the Xoshiro256StarStar algorithm (known as shift-register generators).
+    /// </summary>
+    internal class RandomNumberGenerator
```

Comment:
Is this class copied from some other place? If so could you add a hyperlink where is the original source? It will help to backtrack changes and issues.

Reply:
Added.

Reply:
Resovle